### ISSUE TITLE
[next]Print arrays in the data explorer

### ISSUE THREAD

**Issue by neumino** [rethinkdb/rethinkdb#3672]
For `r.expr([1,2,3])`, the data explorer used to print something like this in the tree view
```
[
  1,
  2,
  3
]
```

Now it just prints
```
1
2
3
```

For r.expr([]) it says that it returns nothing. Nested arrays are still properly displayed.

**Comment by mglukhovsky** [comment]
@danielmewes, @deontologician, @mlucy, this is the bug we spotted in the Data Explorer for arrays. Good catch, @neumino.

**Comment by danielmewes** [comment]
This is actually a 1.16 issue. It seems that arrays are interpreted as if they were sequences. 
Possibly related to @AtnNn Data Explorer changes?

@AtnNn can you take a look at this?  I think the old format is better because it allows people to see the data type that's being returned.

**Comment by AtnNn** [comment]
It is definitely caused by my changes. I think I know how to fix this easily.

**Comment by AtnNn** [comment]
The culprit is

```
if typeof value._next is 'function' # if it's a cursor
```

**Comment by AtnNn** [comment]
Fixed in a9c5a0e

**Comment by AtnNn** [comment]
In coffeescript, `not a instanceof b` is not the same as `a not instanceof b`.
